hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/CPSubsystemInfoCollector.java

```diff
@@ -25,6 +31,34 @@
     @Override
     public void forEachMetric(Node node, BiConsumer<PhoneHomeMetrics, String> metricsConsumer) {
         int cpMemberCount = node.getNodeEngine().getConfig().getCPSubsystemConfig().getCPMemberCount();
-        metricsConsumer.accept(PhoneHomeMetrics.CP_SUBSYSTEM_ENABLED, String.valueOf(cpMemberCount != 0));
+        boolean cpSubsystemEnabled = cpMemberCount != 0;
+        metricsConsumer.accept(PhoneHomeMetrics.CP_SUBSYSTEM_ENABLED, String.valueOf(cpSubsystemEnabled));
+        if (cpSubsystemEnabled) {
+            metricsConsumer.accept(PhoneHomeMetrics.CP_MEMBERS_COUNT, String.valueOf(cpMemberCount));
+
+            RaftService raftService = node.getNodeEngine().getService(RaftService.SERVICE_NAME);
+            int groupsCount = raftService.getMetadataGroupManager().getGroupIds().size();
+            metricsConsumer.accept(PhoneHomeMetrics.CP_GROUPS_COUNT, String.valueOf(groupsCount));
+
+            SemaphoreService semaphoreService = node.getNodeEngine().getService(SemaphoreService.SERVICE_NAME);
+            int semaphoresCount = semaphoreService.getTotalResourcesCount();
```

Comment:
Will this include destroyed semaphores as well? If yes, can we avoid that?

Reply:
Destroyed resources are removed from the registry, so the destroyed semaphores aren't included here